Makefile

```diff
@@ -81,4 +81,7 @@ build_env build_module: guard-STAGING_AWS_ACCOUNT_ID
 	clear_terragrunt_cache \
 	sso_login \
 	lambda \
-	lambdas 
\ No newline at end of file
+	lambdas
+
+glue:
+    @:
```

Comment:
```suggestion
	lambdas
```

Reply:
@ShadeWyrm Will let you review and accept this suggestion. I can approve your PR when it is done as I completed my testing successfully :)